code/LeftAndMain.php

```diff
@@ -226,6 +226,25 @@ class LeftAndMain extends Controller implements PermissionProvider
      */
     private static $extra_requirements_javascript = array();
 
+    /**
+     * Register additional translation sources to be included in the CMS client side code.
+     * This can be use to override SilverStripe translations. This configuration option
+     * expects a list of browser accessible directories that can be pass to
+     * {@link Requirements::add_i18n_javascript()}.
+     *
+     * YAML configuration example:
+     * <code>
+     * LeftAndMain:
+     *   $extra_requirements_i18n_javascript:
+     *     - mysite/client/lang
```

Comment:
```suggestion
     *     - app/client/lang
```

We haven't used mysite for a few minor releases now, may as well not keep documenting it in examples